src/pages/Share/ShareTab.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import {saveUnknownUserDetails} from '@libs/actions/Share';
+import Navigation from '@libs/Navigation/Navigation';
+import MoneyRequestParticipantsSelector from '@pages/iou/request/MoneyRequestParticipantsSelector';
+import CONST from '@src/CONST';
+import ROUTES from '@src/ROUTES';
+
+function ShareTab() {
+    return (
+        <MoneyRequestParticipantsSelector
+            iouType={CONST.IOU.TYPE.SUBMIT}
+            onFinish={() => {}}
```

Comment:
`MoneyRequestParticipantsSelector` is a memoized component, and passing an anonymous function makes it rerender always, because its reference will change